core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/atom/textfield/TextFieldOutlinedPassword.kt

```diff
@@ -129,3 +132,16 @@ private fun selectVisualTransformation(
 }
 
 private fun isShowPasswordAllowed(isEnabled: Boolean, isPasswordVisible: Boolean) = isEnabled && isPasswordVisible
+
+private fun Modifier.applyLegacyPasswordSemantics(): Modifier {
+    /*
+    * Workaround to fix a crash that occurs when the password visibility state changes
+    * and an accessibility service is enabled. It prevents the crash by clearing and
+    * resetting the semantics of the composable.
```

Comment:
The comment needs to be updated to reflect the new code. It should probably also include a link to the relevant Google issue: https://issuetracker.google.com/issues/348232635